evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -344,8 +344,8 @@ def test_make_pipeline_from_components(X_y_binary, logistic_regression_binary_pi
     est = RandomForestClassifier(random_state=7)
```

Comment:
I don't think we have a test anywhere in this file for `compute_estimator_features`. Can you add some for the linear and non-linear case please?